src/components/reusable/breadcrumbs/breadcrumbs.stories.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Kyndryl, Inc. 2023
+ */
+
+// External library imports
+import { html } from 'lit';
+import type { Meta, StoryObj } from '@storybook/web-components';
+
+import '@kyndryl-design-system/foundation/components/link';
+
+// Relative component imports
+import './breadcrumbs';
+import './index';
+
+/**
+ * ## Usage
+ *
+ * Here's how you can use the `kyn-breadcrumbs` component with sample items:
+ *
+ * ```javascript
+ *
+ * html`
+ *  <kyn-breadcrumbs>
+ *    <kd-link class="breadcrumb-item" .href="/">Bridge</kd-link>
+ *    <kd-link class="breadcrumb-item" .href="/catalog">Catalog</kd-link>
+ *    <span class="breadcrumb-item breadcrumb-last">FinOps</span>
+ *  </kyn-breadcrumbs>
+ * `;
+ * ```
```

Comment:
I would avoid using Lit templates in any examples that aren't auto-generated, since the consumer is most likely not using Lit.